docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -264,7 +264,7 @@ The table shows the mappings from Trino to Oracle data types:
     - ``DATE``
     - See :ref:`datetime mapping`
   * - ``TIMESTAMP``
-    - ``TIMESTAMP(3)``
+    - ``TIMESTAMP(6)``
```

Comment:
Since `TIMESTAMP` in trino is an alias for `TIMESTAMP(3) so the precision should be 3 right ?

Reply:
reverted